test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -437,7 +442,8 @@ def test_rnn_init_predict_split(self):
         # predict net. When we embed parameters, there should be more
         # ops in the init net.
         mp = onnx.ModelProto.FromString(do_export(model, input, export_params=self.embed_params,
-                                                  keep_initializers_as_inputs=True)[0])
+                                                  keep_initializers_as_inputs=True,
+                                                  do_constant_folding=False)[0])
```

Comment:
Why do these tests explicitly set `do_constant_folding` to False? Is there bug in constant folding for these tests?

Reply:
No. There's no problem with constant folding here. This test actually matches the number of inputs:
        if self.embed_params:
            assert len(prepared.init_net.op) == 875
            assert len(prepared.predict_net.op) == 130
        else:
            assert len(prepared.init_net.op) == 8
            assert len(prepared.predict_net.op) == 997

I decided not to touch these assertions and leave the test as is.